src/less/filter-menu/base/filter-menu.less

```diff
@@ -0,0 +1,149 @@
+.filter-menu-aria,
+.filter-menu-form {
+    background-color: @filter-menu-item-background-color;
+    border: 1px solid @filter-menu-border-color;
+    border-radius: 16px;
+    display: inline-block;
+    max-width: 288px; // TODO move max-width restriction to filter-menu-button styles
+    min-width: 144px;
+    overflow: hidden;
+}
+
+span.filter-menu-aria,
+span.filter-menu-form {
+    display: inline-block;
+}
+
+.filter-menu-aria__items,
+.filter-menu-form__items {
+    margin-top: 8px;
+    max-height: calc(50vh - 40px); // half of the viewport height || TODO move max-height restriction to filter-menu-button styles
+    min-width: 100%;
+    overflow-y: auto;
+    position: relative;
+    width: auto;
```

Comment:
Isn't the default auto?  Was this overwritten somewhere so we need to reset to auto?

Reply:
That's a good catch. Removed.